recipe-ts-runtime/src/Cake.ts

```diff
@@ -121,4 +121,14 @@ export class Cake {
     public getNamespace(): string {
         return this.prefixStack.join(Cake.SEPARATOR);
     }
+
+    public toJson(): string {
+        return JSON.stringify(this.entries);
+    }
+
+    public static fromJson(jsonCake: string): Cake {
+        const cake = new Cake();
+        cake.entries = JSON.parse(jsonCake);
```

Comment:
isn't `entries` a `Map`? and this assigns a plain `Object`...

Reply:
Yeah, I'm not sure why, but typescript isn't complaining at all.

Reply:
I assume because `JSON.parse` returns `any`...

Reply:
I still think these should be consistent somehow... maybe change the one on line 3?

Reply:
Changed it from a map to an object with string keys.